roles/StackStorm.st2/templates/rbac_mappings/mappings.yml.j2

```diff
@@ -0,0 +1,3 @@
+---
+
+{{ item.mapping | to_nice_yaml(2) }}
```

Comment:
In most of our usage we also pipe to | indent(2) - perhaps that would be useful here too?